web/src/app/schedules/ScheduleDetails.js

```diff
@@ -46,6 +46,16 @@ export default function ScheduleDetails({ scheduleID }) {
   const [configTempSchedule, setConfigTempSchedule] = useState(null)
   const [deleteTempSchedule, setDeleteTempSchedule] = useState(null)
 
+  const onNewTempSched = useMemo(() => () => setConfigTempSchedule(true), [])
+  const onEditTempSched = useMemo(
+    () => (sched) => setConfigTempSchedule(sched),
+    [],
+  )
+  const onDeleteTempSched = useMemo(
+    () => (sched) => setDeleteTempSchedule(sched),
+    [],
+  )
```

Comment:
simplify, also may need to be `handleOnDeleteTempSched` IIRC (linter will say)
```suggestion
  const onDeleteTempSched = useCallback(setDeleteTempSchedule, [])
```